include/deal.II/fe/fe_values.h

```diff
@@ -3508,6 +3508,86 @@ class FESubfaceValues : public FEFaceValuesBase<dim,spacedim>
 #ifndef DOXYGEN
 
 
+// Convenience template functions used in fe_values.cc and its instantiation
+// files
+namespace internal
+{
+  template <class VectorType>
+  typename VectorType::value_type
+  get_vector_element (const VectorType &vector,
```

Comment:
As far as I see, we only use this in `source/fe/fe_values.cc`. Can you elaborate on why you want to move it here?